lib/derivations.nix

```diff
@@ -73,7 +81,7 @@ in
         throwIfNot (derivation.type or null == "derivation")
           "lazySimpleDerivation: input must be a derivation."
           throwIfNot
-          (derivation.outputs == [ "out" ])
+          (derivation.outputs == outputs)
           # Supporting multiple outputs should be a matter of inheriting more attrs.
           "The derivation ${derivation.name or "<unknown>"} has multiple outputs. This is not supported by lazySimpleDerivation yet. Support could be added, and be useful as long as the set of outputs is known in advance, without evaluating the actual derivation."
```

Comment:
This comment needs to be updated, otherwise the `lazyDerivation` change looks good to me. Would be best to have this change in a separate PR.

Reply:
That was quick!
Wanted to do that.
Thanks!

Reply:
Separated into https://github.com/NixOS/nixpkgs/pull/292209

(yes, it's a different PR number)